Clarify restore hook includedResources when not specified (#4490)
vmware-tanzu/velero: site/content/docs/main/api-types/restore.md

      # Array of namespaces to which this hook does not apply. Optional.
      excludedNamespaces:
      - ns3

>       # Array of resources to which this hook applies. If empty, it applies to all supported resources.

Reviewer: ```suggestion
      # Array of resources to which this hook applies. If unspecified, the hook applies to all resources in the backup. Optional.
```
